src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -53,7 +54,7 @@ export class CalcitePickList {
    */
   @Prop({ reflect: true }) mode: "selection" | "configuration" = "selection";
   /**
-   * Multiple works like conventional checkboxes and radio buttons.
+   * Multiple Works similar to standard radio buttons and checkboxes.
```

Comment:
The wording here was different from the valueList. Didn't see a good reason for these to use different language.